data/ai/lua/ai_helper.lua

```diff
@@ -1410,6 +1410,12 @@ function ai_helper.next_hop(unit, x, y, cfg)
     --     path: if given, find the next hop along this path, rather than doing new path finding
     --       In this case, it is assumed that the path is possible, in other words, that cost has been checked
     --     avoid_map: a location set with the hexes the unit is not allowed to step on
+    --     no_fan_out: prior to Wesnoth 1.16, the unit strictly followed the path, which can lead to
```

Comment:
Again, I'd prefer this to be called `fan_out` to avoid dealing in double negatives.